player/scripting.c

```diff
@@ -256,13 +256,18 @@ static int load_cplugin(struct mpv_handle *client, const char *fname)
 {
     int r = -1;
     void *lib = dlopen(fname, RTLD_NOW | RTLD_LOCAL);
-    if (!lib)
+    MPContext *ctx = mp_client_get_core(client);
+    if (!lib) {
+        MP_ERR(ctx, "C plugin library load error: '%s'\n", dlerror());
```

Comment:
Can return NULL, and also this should be called immediately after dlopen, instead of having other stuff in between.